Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -14,21 +14,33 @@ public struct SwiftPackageManagerDependencies: Codable, Equatable {
     // Additional settings to be added to targets generated from SwiftPackageManager.
     public let targetSettings: [String: SettingsDictionary]
 
+    /// The project options.
+    public let options: Project.Options
+
+    /// The resource synthesizers for the project to generate accessors for resources.
+    public let resourceSynthesizers: [ResourceSynthesizer]
```

Comment:
This should be on a package basis, so it should be something like:
```
public let projectConfigurations: [String: ProjectConfiguration]
```

where:
```
struct ProjectConfiguration {
    /// The project options.
    public let options: Project.Options

    /// The resource synthesizers for the project to generate accessors for resources.
    public let resourceSynthesizers: [ResourceSynthesizer]
}
```

And it should be an override for the default ones, only for the packages for which it's specified

Reply:
Ah that makes sense. I will update 👍